tests/unittests/dataset_tests.py

```diff
@@ -1028,6 +1029,20 @@ def test_field_schemas(self):
                 "spam", embedded_doc_type=fo.Detections
             )
 
+    @drop_datasets
+    def test_validate_sample_fields(self):
+        dataset = foz.load_zoo_dataset("quickstart")
```

Comment:
We need to use `fo.Dataset()` here because we don't want unit tests to download datasets from the zoo (too large of a side effect)